contrib/nitiwiki/src/nitiwiki.nit

```diff
@@ -75,7 +75,7 @@ var opt_status = new OptionBool("Display wiki status", "-s", "--status")
 var opt_render = new OptionBool("Render the out directory from markdown sources", "-r", "--render")
 var opt_force = new OptionBool("Force render even if source files are unchanged", "-f", "--force")
 var opt_clean = new OptionBool("Clean the output directory", "--clean")
-var opt_rsync = new OptionBool("Synchronize outdir with distant wiki using rsync", "-s", "--rsync")
```

Comment:
I prefer to use "s" for rsync because the command to build and push stills `nitiwiki -rs` instead of `nitiwiki -r --rsync`.